app/assets/javascripts/app/views/header_view.js

```diff
@@ -6,14 +6,20 @@ app.views.Header = app.views.Base.extend({
 
   className : "dark-header",
 
-  events : {
-    "click ul.dropdown li:first-child" : "toggleDropdown",
+  events :{
+    "click ul.dropdown li:first-child" : "toggleUserDropdown",
     "focusin #q": "toggleSearchActive",
-    "focusout #q": "toggleSearchActive"
+    "focusout #q": "toggleSearchActive",
+    "click #notifications-badge": "toggleNotifDropdown",
```

Comment:
@svbergerem : The `scroll` event seems not to be binded here. Is there a set of default events?